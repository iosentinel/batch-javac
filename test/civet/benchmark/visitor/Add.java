package benchmark.visitor;

public class Add extends BinOp {
	int apply(int x, int y) {
		return x + y;
	}
}
