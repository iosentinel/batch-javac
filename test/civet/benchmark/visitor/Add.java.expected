package benchmark.visitor;

public class Add extends BinOp {
    
    public Add() {
        super();
    }
    
    int apply(int x, int y) {
        return x + y;
    }
    
    static int apply$1005(int x, int y) {
        return x + y;
    }
}